src/ViewHandler/SecuredViewHandler.php

```diff
@@ -0,0 +1,36 @@
+<?php
+declare(strict_types=1);
+
+namespace PhpList\RestBundle\ViewHandler;
+
+use FOS\RestBundle\View\View;
+use FOS\RestBundle\View\ViewHandler;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * This class is used to add headers to the default response.
+ * @author Xheni Myrtaj <xheni@phplist.com> .
+ */
+class SecuredViewHandler
+{
+
+    /**
+     * @param ViewHandler $viewHandler
+     * @param View        $view
+     * @param Request     $request
+     * @param string      $format
+     *
+     * @return Response
+     */
+    public function createResponse(ViewHandler $handler, View $view, Request $request, string $format)
+    {
+        $view->setHeaders([
```

Comment:
Please let's put the `[` and `]` on separate lines. This will avoid formatting changes when using the PhpStorm autoformat or the php-cs-fixer.